parcel_test.go

```diff
@@ -31,57 +31,105 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db") // настройте подключение к БД
+	if err != nil {
+		require.NoError(t, err)
+	}
+	defer db.Close()
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
 	// add
+	newParcel, err := store.Add(parcel)
+	require.NoError(t, err)
+	require.Positive(t, newParcel)
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
 
 	// get
+	geted, err := store.Get(newParcel)
+	require.NoError(t, err)
+	require.Equal(t, parcel.Client, geted.Client)
+	require.Equal(t, parcel.Status, geted.Status)
+	require.Equal(t, parcel.Address, geted.Address)
+	require.Equal(t, parcel.CreatedAt, geted.CreatedAt)
+
 	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
 	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
 
 	// delete
+	err = store.Delete(newParcel)
+	require.NoError(t, err)
+
 	// удалите добавленную посылку, убедитесь в отсутствии ошибки
 	// проверьте, что посылку больше нельзя получить из БД
+	// no get method
+	_, err = store.Get(newParcel)
+	require.Equal(t, sql.ErrNoRows, err)
```

Comment:
Для сравнивания ошибок нужно использовать специализированную функцию
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```